src/Output.test.js

```diff
@@ -71,10 +72,10 @@ test('test2 - renders the correct number of search results as selected by user',
 
 describe('sort function', () => {
   
-    test('test 3: sorts results in alphabetical order if "Title A-Z" sort criteria selected by user', () => {
-        render( 
+    test('test 3: sorts results in alphabetical order if "Title A-Z" sort criteria selected by user', async () => {
+        await render( 
             <Output 
-                sortCriteria="Title" 
+                sortCriteria="TITLE_ASCENDING" 
```

Comment:
you could use `sortMethods.TITLE_ASCENDING.name` here, as using constant variables avoids any potential for typos in the future.